File: java/입문/숫자_찾기.java
class Solution {
    public int solution(int num, int k) {
        String numStr = String.valueOf(num);
        String kStr = String.valueOf(k);
        int answer = numStr.indexOf(kStr) == -1 ? -1 : numStr.indexOf(kStr) + 1;
        
        return answer;
    }
}

File: swift/입문/숫자_찾기.swift
// 숫자 찾기 문제

import Foundation

func solution(_ num:Int, _ k:Int) -> Int {
    if let idx = String(num).map{ Int(String($0)) }.firstIndex(of: k) {
        return idx + 1
    } else {
        return -1
    }
}